IzvorniKod/Views/Shared/_LoginPartial.cshtml

```diff
@@ -5,25 +5,44 @@
 @inject SignInManager<IdentityUser> SignInManager
 @inject UserManager<IdentityUser> UserManager
 
-<ul class="navbar-nav">
+<link rel="stylesheet" href="~/css/header.css" asp-append-version="true">
+
+<ul class="navbar-nav flex-grow-1">
     @if (SignInManager.IsSignedIn(User))
     {
         <li class="nav-item">
-            <a class="nav-link text-dark" asp-area="Identity" asp-page="/Account/Manage/Index" title="Manage">Hello @User.Identity?.Name!</a>
+            <a class="nav-link text-light pe-lg-4 pe-0" asp-area="" asp-controller="Home" asp-action="Chat"><img src="~/images/message-solid.svg" class="header-icon" alt="Icon">Chat</a>
+        </li>
+        <li class="nav-item">
+            <a class="nav-link text-light pe-lg-4 pe-0" asp-area=""><img src="~/images/book-solid.svg" class="header-icon" alt="Icon">View material</a>
+        </li>
+        <li class="nav-item">
+            <a class="nav-link text-light pe-lg- pe-0" asp-area="" asp-controller="Home" asp-action="Calendar"><img src="~/images/calendar-days-solid.svg" class="header-icon" alt="Icon">View calendar</a>
         </li>
+    }
+</ul>
+
+<ul class="navbar-nav">
+    @if (SignInManager.IsSignedIn(User))
+    {
         <li class="nav-item">
-            <form class="form-inline" asp-area="Identity" asp-page="/Account/Logout" asp-route-returnUrl="@Url.Action("Index", "Home", new { area = "" })">
-                <button type="submit" class="nav-link btn btn-link text-dark">Logout</button>
-            </form>
+            <a class="nav-link text-light" asp-area="Identity" asp-page="/Account/Manage/Index" title="Manage">
+                <img src="~/images/user-solid.svg" class="header-icon" alt="Icon">Account management
+            </a>
         </li>
+        @* <li class="nav-item"> *@
+        @*     <form class="form-inline" asp-area="Identity" asp-page="/Account/Logout" asp-route-returnUrl="@Url.Action("Index", "Home", new { area = "" })"> *@
+        @*         <button type="submit" class="nav-link btn btn-link text-light">Logout</button> *@
```

Comment:
ovaj comment vidim da je za logout gumb, ak je odluceno da logout postoji samo u account menagmentu bilo bi dobro removeati unnecesary komentare

Reply:
aha komentirao sam si taj dio koda, zaboravio sam zašto al maknut ću